src/pages/index.astro

```diff
@@ -59,4 +70,55 @@ import Button from "../components/Button.astro";
 </Layout>
 
 <style>
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    color: #fff;
+    font-weight: 600;
+  }
+
+  h1.hero-header {
```

Comment:
Not really sure why, but the font sizes look a bit disproportionate on mobile, the hero text is very big and hits the right border, and the body text feels small. Here it is on iPhone SE dimensions:

![2024-08-05-19-40-56](https://github.com/user-attachments/assets/0986f9b6-e73a-4744-b632-acabcb5caa8a)